examples/advanced/autowrap_ufuncify.py

```diff
@@ -26,9 +26,10 @@
     sys.exit("Cannot import numpy. Exiting.")
 
 import mpmath
+import matplotlib.pyplot as plt
```

Comment:
Since matplotlib is not a sympy dependency, you should do the same thing as numpy above.